unit-test/ds_app_tests.c

```diff
@@ -179,11 +184,11 @@ void DS_AppInitialize_Test_EVSRegisterError(void)
     UtAssert_BOOL_FALSE(OS_ObjectIdDefined(DS_AppData.FileStatus[DS_DEST_FILE_CNT / 2].FileHandle));
     UtAssert_BOOL_FALSE(OS_ObjectIdDefined(DS_AppData.FileStatus[DS_DEST_FILE_CNT - 1].FileHandle));
 
-    /* Note: not verifying that rest of DS_AppData is set to 0, because some elements of DS_AppData
-     * are modified by subfunctions, which we're not testing here */
-    UtAssert_STUB_COUNT(CFE_EVS_SendEvent, 1);
-    UtAssert_INT32_EQ(context_CFE_EVS_SendEvent[0].EventID, DS_INIT_ERR_EID);
-    UtAssert_INT32_EQ(context_CFE_EVS_SendEvent[0].EventType, CFE_EVS_EventType_ERROR);
+    call_count_CFE_EVS_SendEvent = UT_GetStubCount(UT_KEY(CFE_EVS_SendEvent));
```

Comment:
No need to get the stub count of CFE_EVS_SendEvent here that I can see.  Can this be taken out?

Reply:
Thanks @jphickey. It must have been left over from a different idea I had for this test case.
Removed now.